Enterprise-CMCS/macpro-mdct-qmr
# Measurement Date Range Enhancements

### Description



### Related ticket(s)


MDCT-2335

---
### How to test

- Login:
- Start with 2021, open a measure in adult, child, and health home core sets
- Confirm that the "Date Range" section now has a radio button option
- Confirm that when you click no, you see a nested date range (check before and after to see UI difference)
- Repeat for 2022 and 2023

| Before  | After |
| ------------- | ------------- |
| ![image-20230327-145438](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/6752256/12bffe1b-14b9-47fb-97c4-f9368707c273) | ![Screenshot 2023-06-13 at 7 46 04 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/6752256/119f59cf-4655-457f-adf2-5612e5e14463)  |

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
